docs/integrations/builtin/core-nodes/n8n-nodes-base.removeduplicates/index.md

```diff
@@ -0,0 +1,107 @@
+
+* **Disable Dot Notation**: Set whether to use dot notation to reference child fields in the format `parent.child` (turned off) or not (turn on).
+* **Remove Other Fields**: Set whether to remove any fields that aren't used in the comparison (turned on) or not (turned off).
+
+### Remove Items Processed in Previous Executions
+
+When you set the "Operation" field to **Remove Items Processed in Previous Executions**, the Remove Duplicate node compares items in the current input to items from previous executions.
+
+#### Remove Items Processed in Previous Executions parameters
+
+When using the **Remove Items Processed in Previous Executions** operation, the following parameters are available:
+
+* **Keep Items Where**: Select how n8n decides which items to keep. The following options are available:
+	* **Value Is New**: n8n removes items if their value matches items from earlier executions.
+	* **Value Is Higher than Any Previous Value**: n8n removes items if the current value isn't higher than previous values.
+	* **Value Is a Date Later than Any Previous Date**: n8n removes date items if the current date isn't later than previous dates.
+
+* **Value to Dedupe On**: The input field or fields to compare. The option you select for the **Keep Items Where** parameter determines the exact format you need:
+	* When using **Value Is New**, this must be an input field or combination of fields with a unique ID.
+	* When using **Value Is Higher than Any Previous Value**, this must be an input field or combination of fields that has an incremental value.
+	* When using **Value Is a Date Later than Any Previous Date**, this must be an input field that has a date value in ISO format.
+
+#### Remove Items Processed in Previous Executions options
+
+When using the **Remove Items Processed in Previous Executions** operation, the following option is available:
+
+* **Scope**: Sets how n8n stores and uses the deduplication data for comparisons. The following options are available:
+	* **Node**: (default) Stores the data for this node independently from other Remove Duplicates instances in the workflow. When you use this scope, you can [clear the duplication history](#clear-deduplication-history) for this node instance without affecting other nodes.
+	* **Workflow**: Stores the duplication data at the workflow level. This shares duplication data with any other Remove Duplicate nodes set to use "workflow" scope.  n8n will still manage the duplication data for other Remove Duplicate nodes set to "node" scope independently.
+
+When you select **Value Is New** as your **Keep Items Where** choice, this option is also available:
+
+* **History Size**: The number of items for n8n to store to track duplicates across executions. The value of the **Scope** option determines whether this history size is specific to this individual Remove Duplicate node instance or shared with other instances in the workflow. By default, n8n stores 10,000 items.
+
```

Comment:
I have a feeling people will have questions on the exact storage functionality here. Not a blocker.